# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/deva/errno.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/kernel/DEVA_utils.cxx.orig	2005-02-22 15:32:19.278499456 +0100
+++ ./src/kernel/DEVA_utils.cxx	2005-02-22 15:32:36.492882472 +0100
@@ -11,6 +11,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <netdb.h>
+#include <errno.h>
 
 char *DEVA_utils::hostname= NULL;
 char *DEVA_utils::path= NULL;
@@ -206,8 +207,6 @@
     return DEVA_utils::hostname;
 }
 
-extern int errno;
-
 char *DEVA_utils::perror()
 {
     return strerror(errno);
